### DISCUSSION TITLE
[Meeting minutes] [2025-01-23] 회의록

### DISCUSSION THREAD

**Discussion by JaeHyun11** [boostcampaitech7/level4-recsys-finalproject-hackathon-recsys-05-lv3#20] [Meeting]
### 주제

문제 정의 및 실험 방향 설정2

### 진행 상황

공지원: 임베딩 방식 (tf-Idf, Word2Vec BERT, Graph embedding), Auxiliary graph 추가 실험 제안, bought-together 실험은 아마존 데이터셋에서 하면 좋겠다.
김주은: pytorch로 lightGCN 코드 구현 중. split이 없어서 split된 걸 사용해야 됩니다. BPR Loss , 테스트 데이터를 유저별로 똑같이 맞춰야 할 필요는 없는 것 같습니다. train, test split이 오래 걸립니다. sequential로 나눌지, 20%로 나눌지 이야기해요.
류지훈: bpr loss에 대해 설명. negative sampling은 상호작용이 없는 것만 뽑는데, 4점 미만의 영화들 + 유저가 보지 않은 영화들이 후보인데 여기서 유동적으로 비율을 설정할 수 있습니다. 3가지 논문에서 사용한 샘플링, split, loss function, metric 기법, cold-start 평가 방법 설명(long-tail 분포에서 20%/80%로 나눠서 평가)
박세연: GNN을 활용한 사전 학습 방법 - meta aggregator, adaptive neighbor sampler. 샘플링, split, loss function, metric 기법, cold-start 평가(Kshot) 방법 설명
Meta-Learning에 대한 논문 설명. 해당 논문의 split, loss function, metric 기법 설명
박재현: lightGCN, LLM augmenter 논문 읽고 split, loss function, metric 기법 설명
백상민: ligthGCN 코드들을 살펴봤는데 지훈이 코드가 제일 사용하기 좋습니다.
완료된 작업:
진행 중인 작업:

### 직면한 문제점 및 해결 방향

문제점:
해결 방법:
